package.json

```diff
@@ -6,14 +6,14 @@
   },
   "module": "es2022",
   "scripts": {
-    "dev": "node ./src/server/server.js",
-    "start": "NODE_ENV=production node ./src/server/server.js",
-    "debug": "node --inspect src/server/server.js",
-    "debug:prod": "NODE_ENV=production node --inspect src/server/server.js",
-    "local:dev": "nodemon ./src/server/server.js",
-    "local:start": "(export $(grep -v '^#' .env | xargs -d '\n')) && cross-env NODE_ENV=production nodemon ./src/server.js",
-    "local:debug": "nodemon --inspect src/server/server.js",
-    "local:debug:prod": "NODE_ENV=production nodemon --inspect src/server/server.js",
+    "dev": "npm run sass-dev && node ./src/server/server.js",
```

Comment:
should this use sass-dev-watch?

Reply:
The sass-dev-watch task needs to be run separately, as it does not finish and will prevent starting the dev server.